genny/rx/rx.go

```diff
@@ -1,11 +1,13 @@
 package rx
 
 import (
+	"embed"
+
 	"github.com/gobuffalo/genny/v2"
-	packr "github.com/gobuffalo/packr/v2"
 )
 
-var templates = packr.New("github.com/gobuffalo/clara/genny/rx/templates", "../rx/templates")
+//go:embed templates templates/*/*.plush
```

Comment:
As far as I understand from [1], the excluding policy for `.file` and `_file` is only applied when we use directory name format only (without /*), and if we use `*` it really means "everything" as usual. However, for any reason, doing as follow makes an error:

```
//go:embed templates/*
```

I think the reason is that e.g. `templates/buffalo` is matched with the pattern (as a directory name form) but the directory has no content except `_help.plush`. So as a workaround, I wrote the directive as the code I committed.

The test cases have tests for this inclusion indirectly by comparing with the result of generated strings and it worked both for 1.16 and 1.17. (Actually, the rule is the same for 1.16 and 1.17, and I am on 1.16 now. It is a good thing since I do all things under 1.16 and you do on 1.17 :-)

[1] https://pkg.go.dev/embed@go1.16#hdr-Directives

Reply:
Alright, then I'll approve this PR